src/app/navbar/navbar.component.ts

```diff
@@ -1,14 +1,15 @@
 import { Component, OnInit } from '@angular/core';
-
+import {Clock} from '../clock.model';
+declare var require: any;
```

Comment:
1. Przypuszczam, że ta zmienna nie jest używana, czyli nie powinno jej tutaj być
2. Jeśli już mamy potrzebę stworzyć taką zmienną to nie używamy var. 
Odsyłam do lektury: https://medium.com/better-programming/javascript-variables-the-difference-between-var-let-and-const-10efef5d2728